.github/workflows/cargo-update.yml

```diff
@@ -1,6 +1,5 @@
 on:
   workflow_dispatch:
-    branches: [ main ]
```

Comment:
on every branch? Why not add branches explicitly?

Reply:
To prevent issues like this: https://github.com/platten/enarx/actions/runs/2241459166